packages/docs/stories/components/select.stories.ts

```diff
@@ -0,0 +1,404 @@
+/* eslint-disable @typescript-eslint/no-unsafe-argument */
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable import/no-relative-packages */
+import '../../../components/src/components/select/select';
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import {
+  generateScreenshotStory,
+  generateStoryDescription,
+  storybookDefaults,
+  storybookHelpers,
+  storybookTemplate,
+} from '../../src/helpers/component.js';
+
+const { args, argTypes } = storybookDefaults('syn-select');
+const { overrideArgs } = storybookHelpers('syn-select');
+const { generateTemplate } = storybookTemplate('syn-select');
+
+const meta: Meta = {
+  args,
+  argTypes,
+  component: 'syn-select',
+  parameters: {
+    docs: {
+      description: {
+        component: generateStoryDescription('select', 'default'),
+      },
+      story: {
+        height: '250px',
+      },
+    },
+  },
+  title: 'Components/syn-select',
+};
+export default meta;
+
+type Story = StoryObj;
+
+export const Default = {
+  parameters: {
+    args: overrideArgs({
+      name: 'default',
+      type: 'slot',
+      value: `
+        <syn-option value="option-1">Option 1</syn-option>
+        <syn-option value="option-2">Option 2</syn-option>
+        <syn-option value="option-3">Option 3</syn-option>
+      `,
+    }, args),
+    controls: {
+      disable: false,
+    },
+    docs: {
+      description: {
+        story: generateStoryDescription('select', 'default'),
+      },
+    },
+  },
+  render: (renderArgs: unknown) => generateTemplate({ args: renderArgs }),
+} as Story;
+
+export const Focus: Story = {
+  parameters: {
+    chromatic: {
+      disableSnapshot: false,
+    },
+    docs: {
+      description: {
+        story: generateStoryDescription('select', 'focus'),
```

Comment:
The description is currently not shown in Storybook. But I guess it is because the Figma plugin is currently not working again and therefore the token could not be synced